### PR TITLE
add reviewer recommendation to imported comments #2628

### PR DESCRIPTION
**Pull request by kaschioudi**
@asmecher

### CODE REVIEW

File: classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php [hunk 1]
```diff
@@ -217,6 +217,11 @@ function importPeerReviews($args, $request) {
 						$body .= PKPString::stripUnsafeHtml($comment->getComments());
 					}
 				}
+
+				// Add reviewer recommendation
+				$recommendation = $reviewAssignment->getLocalizedRecommendation();
+				$body .= __('reviewer.article.recommendation') . ': ' . $recommendation . "<br>\n";
```
Comment (asmecher) [inline classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php]: This assumes a right-to-left presentation, which might not be appropriate in some languages. I'd suggest adding a new locale key for this.

Comment (kaschioudi) [reply classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php]: Done. https://github.com/pkp/pkp-lib/pull/3096/commits/e6e4209ccd5c6467d2419ca3dc348ee4a9bede24

I came across `submission.recommendation` locale key which I have reused.

**Comment by asmecher** [comment]
Looks good, please go ahead!

**Comment by asmecher** [comment]
(@kaschioudi, just a reminder that this is still on your plate to merge/back-port as needed.)

**Comment by kaschioudi** [comment]
Woops! sorry about that. Merging right away.

**Pull request merged**